pkgs/tools/security/masscan/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, libpcap }:
+
+stdenv.mkDerivation rec {
+  name = "masscan-${version}";
+  version = "2017-02-04";
```

Comment:
This should be the date of the git commit you're packaging, in this case 2016-11-03.

Also, is there a specific reason to package the master branch instead of a release?